.env.example

```
@@ -3,19 +3,6 @@ DISCORD_TOKEN=''
-GEMINI_CODE_EXECUTION_MODEL='gemini-2.0-flash' # Model that executes Python
-
-# Ignore response phrase for AI to signal not to respond
-IGNORE_RESPONSE_PHRASE='~!IGNORE_RESPONSE~|'
-
-# Chance (0.0 to 1.0) that the bot processes a message it wasn't mentioned in.
-# 0.0 = Only responds to mentions and replies.
-# 0.02 = Processes ~2% of all messages (good for cost savings while still feeling "alive").
-# 1.0 = Processes EVERY message (Very costly).
-AI_OVERHEAR_RATE='0.02'
+# AI
+GOOGLE_GENERATIVE_AI_API_KEY=''
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)